.eslintrc.yml

```diff
@@ -1,268 +1,33 @@
 env:
   es6: true
   node: true
-extends: 'eslint:recommended'
+parser: babel-eslint
+extends:
+- eslint:recommended
+- plugin:flowtype/recommended
+plugins:
+- flowtype
 parserOptions:
-  sourceType: module
+  sourceType": module
+  ecmaVersion": 6
+  ecmaFeatures:
+    impliedStrict: true
+    jsx: true
 rules:
```

Comment:
The rules are removed intentionally; lots of them conflict with flow support, so for now just enabled basic style checks, more detailed ones can be added later.